os/windows/cpu-affinity.c

```diff
@@ -323,8 +324,8 @@ int fio_getaffinity(int pid, os_cpu_mask_t *mask)
 		goto err;
 	}
 	if (!GetProcessGroupAffinity(handle, &group_count, current_groups)) {
-		log_err("%s: failed to get single group affinity for pid %d (%d)\n",
-			__func__, pid, GetLastError());
+		log_err("%s: failed to get single group affinity for pid %d "
```

Comment:
In Linux kernel source code it is not allowed to split a format string over multiple lines because that makes it really hard to look up error messages in the source code with grep. I'm not sure whether or not that coding style rule also applies to fio code.

Reply:
Hmm a quick search with
```
ag --cc '%d.*" *$'
```

Does suggest doing the splitting at that point is unusual. I'll fix it up.

Reply:
A slightly different search actually suggests it's pretty common to see wrapped format strings
```
ag --cc '\(.*"$'
```

(above isn't perfect). At any rate I've tried changed the lines I touched.